.obs/specfile/elemental-operator.spec

```diff
@@ -59,6 +59,9 @@ BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 %package -n elemental-register
 Summary: The registration client
 
+Requires: dmidecode
```

Comment:
Not sure about this. Is this really _required_ (in the sense of: program won't work / will crash without) ?
Otherwise it should be a `Recommends` 😉

Reply:
Technically we will fail to register, because the `no-smbios` Registration flag is off by default. 
However it should be possible for users to do a `--no-recommends` install, using the `no-smbios` flag if they need to. So this is probably the best approach.
Thank you.